README.md

```diff
@@ -10,22 +10,18 @@
 
 
    - Day1 
-	   - 9-11: 		Course introduction
-	   - 11-12:30: 		UNIX
-	   - 12:30-13:30: 	Lunch break
-	   - 13:30-18: 		UNIX
-
+	   - 8-10: 		Course introduction
+	   - 10-14:00: 		UNIX
 
    -  Day2
```

Comment:
Everything a little bit squeezed!

Reply:
We can move something from day 2 to day 3! Do it on fly...